lib/resources/Issuing/Cards.js

```diff
@@ -6,14 +6,7 @@ const stripeMethod = StripeResource.method;
 module.exports = StripeResource.extend({
   path: 'issuing/cards',
 
-  includeBasic: [
-    'create',
-    'list',
-    'retrieve',
-    'update',
-    'setMetadata',
-    'getMetadata',
-  ],
+  includeBasic: ['create', 'list', 'retrieve', 'update'],
 
   retrieveDetails: stripeMethod({
```

Comment:
Consistency with other nested resources. `/v1/issuing/cards/{card}/details` only has a single method, but if it had multiple ones they'd be called e.g. `createDetails`.

Arguably the method should be called `listDetails` and not `retrieveDetails` since it doesn't take an ID for the nested resource, but the endpoint returns a single object, not a list object. `card_details` is a bit similar to `balance`, it's sort of a singleton (in the context of a card).

Reply:
In a way, this is not really a nested resource though right? A nested resource would be a POST or a LIST on that endpoint but it's a GET so it's more like `/pay` on Invoice no?

Reply:
@remi-stripe I think it qualifies as a nested resource because the endpoint returns a different resource.

Reply:
Any idea why this is `retrieveDetails` when OpenAPI has it as just `details`?